pkg/features/kube_features.go

```diff
@@ -693,6 +699,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	AnyVolumeDataSource:                            {Default: false, PreRelease: featuregate.Alpha},
 	DefaultPodTopologySpread:                       {Default: false, PreRelease: featuregate.Alpha},
 	SetHostnameAsFQDN:                              {Default: false, PreRelease: featuregate.Alpha},
+	DisableAcceleratorUsageMetrics:                 {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
I don't have strong feelings here. But the same comment applies: https://github.com/kubernetes/kubernetes/pull/91930#discussion_r437685346

Reply:
It's a bit different right? The feature here is "disabling the Accelerators metrics" not "Enabling the accelerator metrics". If it were the latter, then the feature would need to start with the enabled state, which doesn't seem right.

What do you think?

Reply:
IMO the feature is `accelerator usage metrics` and it can be enabled or disabled. :) The above gives a strange "double negative" aka `DisableAcceleratorUsageMetrics == false` is negating a negative and can be prone to misunderstanding. 🤔 I get that the feature is the ability to turn them off I guess?

I can see it both ways, I personally would drop the `Disable` but that is just me.

Reply:
At this stage the prefix Disable make more sense. My point is that in future metrics would be disabled by default and the feature will be to "Enable" them. Dropping prefix all together as @Klaven suggested is another option, like `CollectAcceleratorUsageMetrics` with values `true` or `false`.

By no means it's a blocking comment btw. Naming is hard.

Reply:
The way that I look at it is that it's a a feature flag that is by default disabled, I don't think there's a way around that. 

The flag "AcceleratorUsageMetrics" and "EnableAcceleratorUsageMetrics" are pretty much the same flag and have the same problem where they would need to be enabled at first. I don't really see how we would work around the fact that feature flags are by nature disabled at first.

As you are mentioning above it would make sense if we could set the feature flag to true at first and then switch it to false but then we are going to confuse users because it's not really how other feature flags behave (or feature flags for that matter).